ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/ReplaceReturnTypeTest.fs

```diff
@@ -0,0 +1,28 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type ReplaceReturnTypeTest() =
+    inherit FSharpQuickFixTestBase<ReplaceReturnTypeFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/replaceReturnType"
+
+    [<Test>] member x.``Replace type in Constraint Mismatch``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type in infix application``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type in Sequential``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type in MatchClause``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type with attribute``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type in TryWith``() = x.DoNamedTest()
+    [<Test>] member x.``Replace type in MatchLambda``() = x.DoNamedTest()
```

Comment:
I'd probably drop `Replace type` in all tests, as this is covered by the class name.

I also tend to use a simple name with a number at the end, as often another related test case is added later and it's easier to group them by using a similar name.